editor/components/post-taxonomies/index.js

```diff
@@ -17,18 +16,23 @@ import './style.scss';
 import HierarchicalTermSelector from './hierarchical-term-selector';
 import FlatTermSelector from './flat-term-selector';
 
-export function PostTaxonomies( { postType, taxonomies, taxonomyWrapper = identity } ) {
-	const availableTaxonomies = filter( taxonomies.data, ( taxonomy ) => includes( taxonomy.types, postType ) );
+export function PostTaxonomies( { post, postType, taxonomies, taxonomyWrapper = identity } ) {
+	const availableTaxonomies = filter( taxonomies, ( taxonomy ) => includes( taxonomy.types, postType ) );
 	const visibleTaxonomies = filter( availableTaxonomies, ( taxonomy ) => taxonomy.visibility.show_ui );
 	return visibleTaxonomies.map( ( taxonomy ) => {
 		const TaxonomyComponent = taxonomy.hierarchical ? HierarchicalTermSelector : FlatTermSelector;
+		const restBase = taxonomy.rest_base;
+		const hasCreateAction = get( post, [ '_links', 'wp:action-create-' + restBase ], false );
+		const hasAssignAction = get( post, [ '_links', 'wp:action-assign-' + restBase ], false );
```

Comment:
Cane we move these two variables inside  `withSelect` instead?. Boolean change less often than a objects which avoid rerenders.

Reply:
Oh I guess it's per taxonomy, This is for me an indication that these checks should be moved to the `TaxonomyComponent` instead.

Reply:
Moved in 04aa6e52d